.bingo/Variables.mk

```diff
@@ -33,8 +33,7 @@ DRONE := $(GOBIN)/drone-v1.5.0
 $(DRONE): $(BINGO_DIR)/drone.mod
 	@# Install binary/ries using Go 1.14+ build command. This is using bwplotka/bingo-controlled, separate go module with pinned dependencies.
 	@echo "(re)installing $(GOBIN)/drone-v1.5.0"
-	@# Manual modification: CGo is disabled to make the Drone CLI build successfully on Darwin/arm64 machines.
-	@cd $(BINGO_DIR) && CGO_ENABLED=0 GOWORK=off $(GO) build -mod=mod -modfile=drone.mod -o=$(GOBIN)/drone-v1.5.0 "github.com/drone/drone-cli/drone"
```

Comment:
are you 100% that removing `CGO_ENABLED=0` is ok in this line? I had problems before with go binaries in small images without using this flag

Reply:
Fixed with a commit in this branch, but I'm going to try and land that in a seperate PR and rebase the commit out of this one https://github.com/grafana/grafana/pull/73290